runelite-client/src/main/java/net/runelite/client/plugins/grounditems/GroundItemsPlugin.java

```diff
@@ -164,6 +170,7 @@
 	private final Map<Integer, Color> priceChecks = new LinkedHashMap<>();
 	private LoadingCache<String, Boolean> highlightedItems;
 	private LoadingCache<String, Boolean> hiddenItems;
+	private LoadingCache<String, Boolean> inventoryStacks;
```

Comment:
Caches are only used for hidden/highlighted items so that they support wildcard matching. Since stacked inventory items aren't specified via user input--and thus will not include wildcards--they won't need a cache. Use `inventoryStacksSet` instead.